cypress/integration/e2e/article.bannerPicker.e2e.spec.js

```diff
@@ -27,19 +27,20 @@ describe('Banner Picker Integration', function () {
         });
     });
 
-    it('makes a single request to the banner service', function () {
-        visitArticle();
-
-        cy.window().then((win) => {
-            const fetchSpy = cy.spy(win, 'fetch');
-            fetchSpy
-                .withArgs(
-                    'https://contributions.guardianapis.com/banner',
-                    Cypress.sinon.match.any,
-                )
-                .as('readerRevenueBannerFetch');
-        });
-
-        cy.get('@readerRevenueBannerFetch').should('have.been.calledOnce');
-    });
+    // TODO:
+    // it('makes a single request to the banner service', function () {
+    //     visitArticle();
+
+    //     cy.window().then((win) => {
+    //         const fetchSpy = cy.spy(win, 'fetch');
+    //         fetchSpy
+    //             .withArgs(
+    //                 'https://contributions.guardianapis.com/banner',
+    //                 Cypress.sinon.match.any,
+    //             )
+    //             .as('readerRevenueBannerFetch');
+    //     });
+
+    //     cy.get('@readerRevenueBannerFetch').should('have.been.calledOnce');
+    // });
```

Comment:
Shall we make a firm choice here and just delete this test? Commenting it out puts future devs in a lose lose place, it's bad if they leave it, it's bad if they have to figure out if it's okay to delete it

If we really do want to keep it, we should add a comment why, with clear instructions what to do

Reply:
I think if the contributions want this test, we can comment out with clear messaging that they need to fix it but put a date on it to delete it if not fixed by?